packages/libsolv/libsolv.spec

```diff
@@ -232,10 +235,16 @@ set -ex
   -DPYTHON_EXECUTABLE=%{__python2}               \
 %if %{with python3_bindings}
   -DENABLE_PYTHON3=ON                            \
-  -DPYTHON3_EXECUTABLE=%{__python3}              \
+  -DPYTHON_EXECUTABLE=/usr/bin/python3.11        \
+  -DPYTHON_LIBRARY=/usr/lib64/libpython3.11.so.1.0 \
```

Comment:
I don't see `PYTHON_LIBRARY` in the libsolv source code at all. Why is this needed?

Reply:
If I don't add this, it builds with python 3.9 on EL9

Reply:
Is it not enough to just switch -DPYTHON3_EXECUTABLE=/usr/bin/python3.11 ??  And does it still break even though we are resetting these macros at the beginning?

Reply:
and PYTHON_LIBRARY appears to be a function of CMAKE itself, helping specify which python library to use.  Probably useful with code generation with swig https://cmake.org/cmake/help/latest/module/FindPythonLibs.html